board/app/controllers/user_controller.php

```diff
@@ -1,29 +1,30 @@
 <?php
 
-class UserController extends AppController{
+class UserController extends AppController {
+
+const MAX_DATA_PER_PAGE = 5;
```

Comment:
You can do either of the following:
- define this in your core.php since I noticed that it is being used in all your paginations for user/comments/threads
- define this in your SimplePagination class
- rename all const variable names per class accordingly ( MAX_COMMENT_PER_PAGE for comments, MAX_USERS_PER_PAGE for users, etc...)
:bow: